LiveSplit.ActionHenk.asl

```diff
@@ -205,6 +211,20 @@ split {
         current.last_level == vars.credits_level_id &&
         current.game_state == vars.state_post_game) return true;
   }
+  else if(settings["category_all_levels"]) {
+    // Splits are based on the number of batches marked as complete
+    return vars.old_batches_completed == 0  && vars.new_batches_completed == 1  ||
+           vars.old_batches_completed == 1  && vars.new_batches_completed == 2  ||
+           vars.old_batches_completed == 2  && vars.new_batches_completed == 3  ||
+           vars.old_batches_completed == 3  && vars.new_batches_completed == 4  ||
+           vars.old_batches_completed == 4  && vars.new_batches_completed == 5  ||
+           vars.old_batches_completed == 5  && vars.new_batches_completed == 6  ||
+           vars.old_batches_completed == 6  && vars.new_batches_completed == 7  ||
+           vars.old_batches_completed == 7  && vars.new_batches_completed == 8  ||
+           vars.old_batches_completed == 8  && vars.new_batches_completed == 9  ||
+           vars.old_batches_completed == 9  && vars.new_batches_completed == 10 ||
+           vars.old_batches_completed == 10 && vars.new_batches_completed == 11;
```

Comment:
`vars.new_batches_completed == vars.new_batches_completed + 1;` ?

Reply:
I'm sure you mean `vars.new_batches_completed == vars.old_batches_completed + 1;`, but yeah idk how I didn't think of that.

Reply:
Alright, I fixed this.